client/src/Theme.jsx

```diff
@@ -1,21 +1,81 @@
 import React from 'react';
-import { ThemeProvider } from 'styled-components';
+import { createGlobalStyle } from 'styled-components';
 
-const theme = {
-  colors: {
-    powderWhite: '#FFFDF9',
-    persianGreen: '#06B49A',
-    lightBlue: '#AFDBD2',
-    onyx: '#36313D',
+export const GlobalStyle = createGlobalStyle`
+  body {
+  background: white;
+  color: black;
+  /* background-color: ${({ theme }) => theme.color.purple}; */
```

Comment:
Are we planning on using the commented-out code? We would want to remove what we don't use before merging to main.

Reply:
Some, yes - leaving as an option. But removed most of it/ what we don't need reference to